pyvo/dal/tests/test_sia2_remote.py

```diff
@@ -254,3 +255,12 @@ def test_res_format(self):
         for rr in results:
             assert rr.access_format == \
                 'application/x-votable+xml;content=datalink'
+
+    @pytest.mark.filterwarnings("ignore::pyvo.dal.exceptions.DALOverflowWarning")
+    def test_reg_sia2(self):
+        image_services = regsearch(servicetype='sia2')
+        irsa_seip = \
+            [s for s in image_services if
+             'irsa' in s.ivoid and 'seip' in s.ivoid][0]
+        result = irsa_seip.search(pos=(2.8425, 74.4846, 10), maxrec=1)
```

Comment:
Minor comment to have a smaller query

```suggestion
        result = irsa_seip.search(pos=(31.8425, 77.4846, 0.1), maxrec=1)
```